extractor/src/test/suite/scala/com/digitalasset/extractor/TlsSpec.scala

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.extractor
+
+import java.io.File
+import java.nio.file.{Files, Path}
+
+import com.digitalasset.daml.bazeltools.BazelRunfiles._
+import com.digitalasset.extractor.config.SnapshotEndSetting
+import com.digitalasset.extractor.services.ExtractorFixture
+import com.digitalasset.extractor.targets.TextPrintTarget
+import com.digitalasset.ledger.api.testing.utils.SuiteResourceManagementAroundAll
+import com.digitalasset.ledger.api.tls.TlsConfiguration
+import com.digitalasset.platform.sandbox.config.SandboxConfig
+import org.apache.commons.io.FileUtils
+import org.scalatest._
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+@SuppressWarnings(Array("org.wartremover.warts.Any"))
+class TlsSpec
+    extends FlatSpec
+    with Suite
+    with SuiteResourceManagementAroundAll
+    with ExtractorFixture
+    with Matchers {
+
+  override protected def darFile = new File(rlocation("extractor/VeryLargeArchive.dar"))
+
+  private def extractCerts: Path = {
+    val dir = Files.createTempDirectory("TlsIT").toFile
+    dir.deleteOnExit()
+    List("server.crt", "server.pem", "ca.crt", "client.crt", "client.pem").foreach { src =>
+      val target = new File(dir, src)
+      target.deleteOnExit()
+      val stream = getClass.getClassLoader.getResourceAsStream("certificates/" + src)
+      FileUtils.copyInputStreamToFile(stream, target)
+    }
+    dir.toPath
+  }
```

Comment:
Sticking to `Path` (except for `.deleteOnExit`) might make it easier to avoid Apache Commons here.

```scala
  private def extractCerts: Path = {
    val dir = Files.createTempDirectory("TlsIT")
    dir.toFile.deleteOnExit()
    List("server.crt", "server.pem", "ca.crt", "client.crt", "client.pem").foreach { src =>
      val target = path.resolve(src)
      target.toFile.deleteOnExit()
      val stream = getClass.getClassLoader.getResourceAsStream("certificates/" + src)
      Files.copy(stream, target)
    }
    dir
  }
```

Reply:
Nice, thanks.